### PR TITLE
feat(clients): add bigquery to sourceTypes enum

### PR DESCRIPTION
**Pull request by damcou**
## 🧭 What and Why

🎟 JIRA Ticket: DI-1034

### Changes included:

- added `bigquery` to `sourceTypes` enum

### CODE REVIEW

**Comment by millotp** [comment]
you need to add the `SourceBigQuery` alongside

**Comment by damcou** [comment]
> you need to add the `SourceBigQuery` alongside

Here you go

File: specs/ingestion/common/schemas/source.yml [hunk 1]
```diff
@@ -200,12 +200,36 @@ SourceCSV:
...
+  properties:
+    projectID:
+      type: string
+      description: Project ID of the BigQuery Source.
+    datasetID:
+      type: string
+      description: Dataset ID of the BigQuery Source.
+    dataType:
+      type: string
+      enum: ['ga4', 'ga360']
```
Comment (millotp) [inline specs/ingestion/common/schemas/source.yml:215]: I think you need to inline this enum, or you're going to upset a linter :)

Comment (damcou) [reply specs/ingestion/common/schemas/source.yml]: fixed

File: specs/ingestion/common/schemas/source.yml [hunk 2]
```diff
@@ -200,12 +200,39 @@ SourceCSV:
...
+      type: string
+      description: Table name (for default BQ).
+    tablePrefix:
+      type: string
+      description: Table prefix (for Google Analytics).
+  required:
+    - projectID
+    - datasetID  
+
+DataType:
```
Comment (millotp) [inline specs/ingestion/common/schemas/source.yml:225]: I'm gonna be a relou but all objects are under the same scope so not easy to find with generic names, can you rename this to `BigQueryDataType` or something pls ?

Comment (damcou) [reply specs/ingestion/common/schemas/source.yml]: fixed, little relou boy

**Review by millotp — APPROVED** [review]
looks good !

**Pull request merged**